src/main/app/src/app/core/service/util.service.ts

```diff
@@ -1,36 +1,21 @@
 /*
- * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-FileCopyrightText: 2024 Lifely
```

Comment:
This is not correct! We need to mention Atos here. Please see our CONTRIBUTING.md

Reply:
ok; read!

changed into `SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely`